at/delete_jobs.cgi

```diff
@@ -1,16 +1,19 @@
 #!/usr/local/bin/perl
 # Delete multiple at jobs
+use strict;
+use warnings;
+our(%access, %text, %in);
 
 require './at-lib.pl';
 &ReadParse();
 &error_setup($text{'deletes_err'});
-@d = split(/\0/, $in{'d'});
+my @d = split(/\0/, $in{'d'});
 @d || &error($text{'deletes_enone'});
 
 # Delete each one
-@jobs = &list_atjobs();
-foreach $d (@d) {
-	($job) = grep { $_->{'id'} eq $d } @jobs;
+my @jobs = &list_atjobs();
+foreach my $d (@d) {
+	my ($job) = grep { $_->{'id'} eq $d } @jobs;
```

Comment:
No, I think that would cause $job to be assigned the number of matches (typically 1 or 0)